QHyper/converter.py

```diff
@@ -13,6 +13,9 @@
 from QHyper.problems.base import Problem
 import numpy as np
 
+def dict_to_list(my_dict) -> list[tuple[Any, ...]]:
+    return [tuple([*key, val]) for key, val in my_dict.items()]
+
```

Comment:
are you using this anywhere?

Reply:
It should be used in `to_cqm()`, but at this point this method does not work as expected, so it is not needed. For now I will remove this method and write again if needed in the future